rotkehlchen/constants/timing.py

```diff
@@ -21,3 +21,4 @@
 ENS_AVATARS_REFRESH = DAY_IN_SECONDS
 SPAM_ASSETS_DETECTION_REFRESH = HOUR_IN_SECONDS * 6
 AUGMENTED_SPAM_ASSETS_DETECTION_REFRESH = DAY_IN_SECONDS
+OWNED_ASSETS_UPDATE = HOUR_IN_SECONDS * 2
```

Comment:
Can also be every hour. Is fine. It takes very little time, right? Just one sqlite query

Reply:
yes shouldn't take much. Is more: one query per table + the insert but is fast